core/templates/dev/head/collection_editor/CollectionSkillsListDirective.js

```diff
@@ -0,0 +1,30 @@
+// Copyright 2015 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for creating a list of skills (acquired and
+ * pre-requisite skills) associated with a collection.
+ *
+ * @author mgowano@google.com (Abraham Mgowano)
+ */
+
+oppia.directive('collectionSkillsListDirective', [function() {
+  return {
+    restrict: 'E',
+    scope: {
+      collectionSkills : "=collectionSkills"
```

Comment:
We probably don't need two-way binding here. Probably better to err on the side of keeping it read-only so it isn't accidentally changed in the future. It can be changed back to two-way if it does need to be changed later. Also, read-only bindings should be named like a getter, e.g.:

```javascript
collectionSkills: '&getCollectionSkills'
```

There was also a minor typo before the colon and strings should be single quoted; both of these were fixed in the above snippet. Note that to access this, you need to use function syntax (I added comments in the directive's HTML file for this).